src/stdlib/String.ts

```diff
@@ -144,11 +144,11 @@ export const Mid = new Callable(
 );
 
 /**
- * Return a string from a float. If it is positive, prefix it with a space.
+ * Return a string from a float/double. If it is positive, prefix it with a space.
  */
 export const Str = new Callable("Str", {
     signature: {
-        args: [new StdlibArgument("value", ValueKind.Float)],
+        args: [new StdlibArgument("value", ValueKind.Double)],
```

Comment:
So this function will accept both doubles and floats? Do you happen to know where the logic is where we accept `ValueKind.Float` for signatures that require `ValueKind.Double`?

Reply:
Good question. Could be wrong, but it's either this code or very similar to it:
https://github.com/sjbarag/brs/blob/main/src/interpreter/index.ts#L453
```
        ... before doing assignment
        if (requiredType && requiredType !== value.kind) {
            if (requiredType === ValueKind.Int64 && value.kind === ValueKind.Int32) {
                value = new Int64(value.getValue());
            } else if (requiredType === ValueKind.Double && value.kind === ValueKind.Float) {
                value = new Double(value.getValue());
       ... now do assignment
```